xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlaySSA.h

```diff
@@ -34,7 +34,6 @@ class CDVDOverlaySSA : public CDVDOverlay
   {
     replace = true;
     m_libass = libass;
-    libass->Acquire();
```

Comment:
@popcornmix 
https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleParserSSA.cpp#L57

this looks wrong. the instance is already create with a ref count of 1